src/main/java/app/controller/comment/CommentController.java

```diff
@@ -0,0 +1,99 @@
+package app.controller.comment;
+
+import app.dto.comment.CommentRequestDto;
+import app.dto.comment.CommentResponseDto;
+import app.service.comment.CommentService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Comment management", description = "Endpoints for managing comments")
+@RestController
+@RequestMapping(value = "/api/comments")
+@RequiredArgsConstructor
+public class CommentController {
+    private final CommentService commentService;
+
+    @PostMapping
+    @PreAuthorize("hasRole('USER')")
+    @ResponseStatus(HttpStatus.CREATED)
+    @Operation(
+            summary = "Add a new comment",
+            description = "Add a new comment to a task by a user"
+    )
+    public CommentResponseDto save(
+            @RequestParam Long userId,
+            @RequestParam Long taskId,
+            @RequestBody @Valid CommentRequestDto commentRequestDto
+    ) {
+        return commentService.save(commentRequestDto, userId, taskId);
+    }
+
+    @GetMapping
+    @PreAuthorize("hasRole('USER')")
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
I don't think this controller will return any other status than OK 200 in case of success. I guess.